.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [12, 14, 16, 18, 20]
```

Comment:
If we drop support for Node.js v12 we have to do a major release

Reply:
yep it makes sense